lvmd/proto/lvmd.proto

```diff
@@ -76,7 +76,8 @@ message WatchResponse {
 
 message WatchItem {
     uint64 free_bytes = 1;  // Free space of the volume group in bytes.
-    string device_class = 2;
+    uint64 size_bytes = 2;  // Size of the volume group in bytes.
+    string device_class = 3;
```

Comment:
Adding a new field between old fields breaks backward compatibility.
The new field should be placed last, IMO.

Reply:
Right.  Protocol buffer keeps backward-compatibility
by not changing the index of fields.

Reply:
Sorry... I fixed it.